src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FinalLocalVariableCheck.java

```diff
@@ -456,6 +456,7 @@ private void storePrevScopeUninitializedVariableData() {
      * @param prevScopeUninitializedVariableData variable for previous stack of uninitialized
      *     variables
      * @noinspection MethodParameterNamingConvention
+     * @noinspectionreason complicated check requires descriptive naming
```

Comment:
MethodParameterNamingConvention 
name is missed

Reply:
Thanks, done.